test/runtests.jl

```diff
@@ -78,13 +102,18 @@ Qf = Qf_rand * transpose(Qf_rand) + I
 
 mulbf    = build_QP_JuMP_model(Q,R,A,B, N;s0 = s0, sl = sl, su = su, ul = ul, uu = uu, Qf = Qf)
 dnlpulbf = LQDynamicData(s0, A, B, Q, R, N; sl = sl, su = su, ul = ul, uu = uu, Qf = Qf)
-lqpulbf  = LQDynamicModel(dnlpulbf)
+lqpulbfs  = LQDynamicModel(dnlpulbf;condense=false)
+lqpulbfc  = LQDynamicModel(dnlpulbf;condense=true)
```

Comment:
The two names are a bit difficult to read ... how about using simply `lq_original` and `lq_condensed`?